### PR TITLE
Debian9

### PR DESCRIPTION
**Pull request by jburel**
Add installation instruction for debian9

This is the first step towards integration in our installation doc
https://trello.com/c/00CYQi2t/29-debian-installation-page

cc @mtbc

### CODE REVIEW

File: linux/step01_debian9_pg_deps.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PGVER=${PGVER:-pg96}
+
+# Postgres installation
+# 9.6
+#start-recommended
+apt-get -y install postgresql
```
Comment (mtbc) [inline linux/step01_debian9_pg_deps.sh]: maybe `postgresql-9.6` to match the comment?

Comment (jburel) [reply linux/step01_debian9_pg_deps.sh]: this is the version installed when running ``postgresql``
in other installations doc we have not been explicit unless we have to

Comment (mtbc) [reply linux/step01_debian9_pg_deps.sh]: Ah, that's fine: it's unlikely to change until buster is released, stable-updates alone wouldn't usually change that, I just wasn't sure if the "9.6" comment meant "this is probably going to be 9.6" or "we actually want 9.6", sounds like the former was the thinking. :+1:

Comment (jburel) [reply linux/step01_debian9_pg_deps.sh]: I will clarify the comment.
Thanks

**Comment by dominikl** [comment]
Haven't reviewed any of the code yet, but a docker build went fine, OMERO 5.3.3 seems to be happy running on debian9 with this PR 👍

**Comment by jburel** [comment]
@dominikl ok to merge that PR so we can check the doc autogeneration?

**Comment by dominikl** [comment]
👍 Yes, good to merge.

**Comment by jburel** [comment]
Thanks

**Pull request merged**